Android/app/src/main/java/fashiome/android/activities/HomeActivity.java

```diff
@@ -98,7 +92,31 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         switch (requestCode) {
 
             case 100:
-                mProductsFragment.addNewProductToList((Product) data.getParcelableExtra("product"));
+
```

Comment:
Adding the fragment to the list is not correct! It is not a timeline that the feed will remain constant. Based on search result or where the user is (geo location) the result set will change.